shipyard_blueprints/ftp/shipyard_ftp/cli/download.py

```diff
@@ -171,126 +58,88 @@ def find_matching_files(file_names, file_name_re):
             )
 
-        matching_file_names = find_matching_files(files, re.compile(source_file_name))
-
-        number_of_matches = len(matching_file_names)
+            number_of_matches = len(matching_file_names)
 
-        if number_of_matches == 0:
-            print(f'No matches were found for regex "{source_file_name}".')
-            sys.exit(EXIT_CODE_NO_MATCHES_FOUND)
+            if number_of_matches == 0:
+                logger.info(f'No matches were found for regex "{source_file_name}".')
+                sys.exit(EXIT_CODE_NO_MATCHES_FOUND)
 
-        print(f"{len(matching_file_names)} files found. Preparing to download...")
-
-        for index, file_name in enumerate(matching_file_names):
-            destination_name = determine_destination_name(
-                destination_folder_name=destination_folder_name,
-                destination_file_name=args.destination_file_name,
-                source_full_path=file_name,
-                file_number=index + 1,
+            logger.info(
+                f"{len(matching_file_names)} files found. Preparing to download..."
             )
 
-            print(f"Downloading file {index + 1} of {len(matching_file_names)}")
-            try:
-                download_ftp_file(
-                    client=client,
-                    file_name=file_name,
-                    destination_file_name=destination_name,
+            for index, file_name in enumerate(matching_file_names):
+                destination_name = shipyard.determine_destination_full_path(
+                    destination_folder_name=destination_folder_name,
+                    destination_file_name=args.destination_file_name,
+                    source_full_path=file_name,
+                    file_number=index + 1,
                 )
-            except Exception as e:
-                print(f"Failed to download {file_name} due to {e}... Skipping")
-    else:
-        destination_name = determine_destination_name(
-            destination_folder_name=destination_folder_name,
-            destination_file_name=args.destination_file_name,
-            source_full_path=source_full_path,
-        )
 
-        try:
-            download_ftp_file(
-                client=client,
-                file_name=source_full_path,
-                destination_file_name=destination_name,
-            )
-        except Exception as e:
-            print(f"The server says: {e}")
-            print(
-                "Most likely, the file name/folder name you specified has typos or the full folder name was not provided. Check these and try again."
-            )
-            sys.exit(EXIT_CODE_NO_MATCHES_FOUND)
+                logger.info(
+                    f"Downloading file {index + 1} of {len(matching_file_names)}"
+                )
 
-    print("Download process complete.")
+                client.download(file_name, destination_name)
+
+        logger.info("Download process complete.")
```

Comment:
I think it is useful to see in the logs where the file is being downloaded to. What about something like:
`logger.info('Downloaded file to {destination_path}')`. Again, not a requirement, but I think that provides more useful information than "Download process complete"